bin/infrasim-init

```diff
@@ -95,6 +89,7 @@ def prepare_libraries():
     run_command("cd /usr/local/lib && ln -sf libOpenIPMIutils.so.0.0.1 libOpenIPMIutils.so")
     run_command("ldconfig")
 
+
 def prepare_seabios():
```

Comment:
prepare_binaries() might be better and gives a place for other binary preparation.